src/utils/EventHandlers.tsx

```diff
@@ -76,20 +77,35 @@ export const handleMarkerOnMouse = (
 
 export const handleMarkerOnClick = (
```

Comment:
Related to "Clicked table row triggers highlight/details/scroll only for that row and not its siblings"